src/data/program_configuration.js

```diff
@@ -650,7 +650,7 @@ export default class ProgramConfiguration {
     defines(): Array<string> {
         const result = [];
         for (const property in this.binders) {
-            result.push.apply(result, this.binders[property].defines());
+            result.push(...this.binders[property].defines());
```

Comment:
I don't think IE supports the spread operator – is this getting transpiled this now?

Reply:
Yes: https://buble.surge.sh/#foo.bar(...args)